nss-cohort-36/bangazon-workforce-mgt-lords-of-might-magic
# Sp computer employee form

# Description

You can now assign a computer when you create it!

Closes #[Enter issue #]

## Type of change

Please delete options that are not relevant.



# Testing Instructions

Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout sp-computer-employee-form`
4. create a new computer and assign it to an employee (make sure that you first have some employees that do not have computers assigned)

# Checklist:
